public/basket.html

```diff
@@ -334,26 +334,27 @@ <h4 class="modal-title" id="credit-card">Credit Card</h4>
         });
     }
 
-    function updateCart() {
-        console.log('Updating cart.');
-        $("tr.item").each(function (){
-            var id = $(this).find("input.id").val(),
-                quantity = $(this).find("input.form-control").val();
-            console.log("Item nr " + id  + " has " + quantity);
-            $.ajax({
-                url: "cart/" + id,
-                type: "PUT",
-                data: JSON.stringify({"quantity": quantity}),
-                success: function (data, textStatus, jqXHR) {
-                    console.log('Worked!');
-                    location.reload();
-                },
-                error: function (jqXHR, textStatus, errorThrown) {
-                    console.error('Could not update item: ' + id + ', due to: ' + textStatus + ' | ' + errorThrown);
-                }
-            });
-        });
-    }
+    // function updateCart()
+	// for each item row in cart table call updateToCart (client.js - updateToCart(itemId, quantity, callback))
```

Comment:
The comment says foreach, but this code isn't wrapped in a loop.

Also, the `next` function defined below is only called once. So it can be inlined.